runtime/elem/builtins/Core.h

```diff
@@ -49,13 +36,30 @@ namespace elem
                 if (!val.isBool())
                     return ReturnCode::InvalidPropertyType();
 
-                targetGain.store(FloatType(val ? 1 : 0));
+                if (val)
+                    fade.fadeIn();
+                else
+                    fade.fadeOut();
             }
 
             if (key == "channel") {
                 channelIndex.store(static_cast<int>((js::Number) val));
             }
 
+            if (key == "fadeInMs") {
+                if (!val.isNumber())
+                    return ReturnCode::InvalidPropertyType();
+
+                fade.setFadeInTimeMs(GraphNode<FloatType>::getSampleRate(), static_cast<float>((js::Number) val));
```

Comment:
Super nitpick but you don't need the `static_cast<float>` because `(js::Number)` is just a `double`, and the `setFadeInTimeMs` and equivalent methods below accept double